pkg/signer/signer.go

```diff
@@ -18,20 +18,27 @@ package signer
 
 import (
 	"context"
+	"crypto/x509"
 	"fmt"
 	"strings"
 
 	"github.com/sigstore/sigstore/pkg/kms/gcp"
 	"github.com/sigstore/sigstore/pkg/signature"
 )
 
-func New(ctx context.Context, signer string) (signature.Signer, error) {
+func New(ctx context.Context, signer string) (signature.Signer, []*x509.Certificate, error) {
 	switch {
 	case strings.HasPrefix(signer, gcp.ReferenceScheme):
-		return gcp.NewGCP(ctx, signer)
+		kms, err := gcp.NewGCP(ctx, signer)
+		// TODO: Add timestamping cert issued by Fulcio and Fulcio's Root CA.
```

Comment:
I feel like we're intermingling too many things on this API... if the timestamp cert is imported (and not part of the interaction b/t Rekor and GCP KMS, this feels a bit awkward...

Reply:
My other thought was to explicitly request one, but you would need provide the CA